packages/satori-js/api.gen.ts

```diff
@@ -390,7 +394,8 @@ export class SatoriApi {
 }
 
   /** List all available flags for this identity. */
-  satoriGetFlags(bearerToken: string,
+  satoriGetFlags(bearerToken: string,basicAuthUsername: string,
```

Comment:
spacing here between params

Reply:
Fixed but have a feeling this is a weird whitespace quirk of the codegen cos there's space in the template. 😅